src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php

```diff
@@ -21,12 +21,9 @@
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
-use Surfnet\ServiceProviderDashboard\Domain\Entity\PrivacyQuestions;
```

Comment:
Not sure why this is gone, was it obsolete already?

Reply:
That was a useless import statement, the PrivacyQuestions are in the same namespace. So importing them is not usefull.